algebra/tests/matrix/buffs.h

```diff
@@ -139,3 +139,44 @@ static const float buffs_EtimesF[] = {
 };
 
 #endif
+/* Matrix F with B inscribed on row 0 and column 0 */
+static const unsigned int buffs_rowsFwithBonRow0Col0 = 15, buffs_colsFwithBonRow0Col0 = 12;
+
+static const float buffs_FwithBonRow0Col0[] = { 
```

Comment:
```suggestion
static const float buffs_BintoF_R0C0[] = { 
```
little convention break, but much more readable